collectors/cgroups.plugin/README.md

```diff
@@ -147,6 +147,33 @@ Support per distribution:
 |AMI|NO|[here](http://pastebin.com/FrxmptjL)|not a systemd system|
 |CentOS 7.3.1611|NO|[here](http://pastebin.com/SpzgezAg)|can be enabled, see below|
 
+### Monitored systemd service metrics
+
+- CPU utilization
+- Used Memory
+- RSS Memory
+- Mapped Memory
+- Cache Memory
+- Writeback Memory
+- Memory Minor Page Faults
+- Memory Major Page Faults
```

Comment:
It's a nitpick, but I would prefer that lists of metrics follow sentence case capitalization, where only the first word is capitalized. For example:

```suggestion
- Used memory
- RSS memory
- Mapped memory
- Cache memory
- Writeback memory
- Memory major page faults
- Memory major page faults
```

Reply:
Done.